chia-blockchain/tasks/main.yml

```
@@ -126,3 +126,11 @@
     dest: /etc/vector/chia-blockchain.vector.toml
   when: add_vector_config
   notify: restart-vector
+
+- name: Add vector config
+  become: yes
+  ansible.builtin.template:
+    src: monit_config.j2
+    dest: /etc/monit/conf.d/
```

```suggestion
    dest: /etc/monit/conf.d/chia-monit
```

Suggest calling it a specific file here - monit_config is probably too generic for the destination server if other things put configs here also